Makefile

```diff
@@ -54,14 +54,22 @@ test-integration-docker:
 ###                                DockerNet                                ###
 ###############################################################################
 
-init-docker:
-	sh scripts/init_main.sh -${build}
+build-docker: 
+	@sh scripts/build.sh -${build} ${BUILDDIR}
+	
+start-docker: build-docker
```

Comment:
Should start-docker run stop-docker as its first command? Seems like `start_network.sh` runs `docker-compose down` under the hood, so maybe already handled. Flagging just in case!

Reply:
Good call!